### ISSUE TITLE
V2-Beta Register

### ISSUE THREAD

**Issue by LifeGold** [WhiteCoreSim/WhiteCore-Webui#3]
Downloaded an updated version of the website, when registration will issue this.
 http://prntscr.com/a849ry http://prntscr.com/a84a2p

**Comment by WhiteCoreWebUI** [comment]
yes i have not uploaded all SQL files which are needed for this yet as i am preparing them for public use also i dont recommend using this version yet as it is highly unstable and needs a lot of improving

**Comment by WhiteCoreWebUI** [comment]
Also it might help if you tell me if you edited the mysql.php and config.php and post the entries without passwords

**Comment by LifeGold** [comment]
mysql.php and config.php and some files where was the connection with the MySQL Password edited is the same as was set in configuration files on server files.
